assets/js/state/sagas/sapSystems.js

```diff
@@ -64,6 +66,16 @@ function* applicationInstanceRegistered({ payload }) {
   );
 }
 
+export function* applicationInstanceMoved({ payload }) {
+  yield put(updateApplicationInstanceHost(payload));
+  yield put(
+    notify({
+      text: `The application instance ${payload.instance_number} has been moved.`,
```

Comment:
Could we have `The application instance ${instance_number} in ${sid} has been moved`